pcsx2/gui/MainFrame.cpp

```diff
@@ -665,7 +665,7 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	menubar.Enable( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
+	menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
```

Comment:
Are you sure it's only the GS plugin that still reports it's initialised? It could be that all the plugins have not been shutdown yet.